### DISCUSSION TITLE
Pico rtc hiccup

### DISCUSSION THREAD

**Discussion by kjm1102** [micropython/micropython#9985] [Hardware & Peripherals]
Needed to know day-of-the week on the pico so tried 
```
>>> gmtime()
(2022, 11, 16, 14, 12, 45, 1, 320)
```
Incorrect, today is Wednesday which is a 2 not a one. So I tried
```
>>> gmtime(time())
(2022, 11, 16, 14, 12, 58, 2, 320)
```
Correct. Just add it to the list of upython stuff that needs fixin' on the pico. If only my skills as a programmer matched my dislike of ugly little gotchas like this.

**Comment by scruss** [comment]
Seems to do the same on a pyboard:
```
MicroPython v1.19.1 on 2022-06-18; PYBv1.1 with STM32F405RG
Type "help()" for more information.
>>> import time
>>> time.gmtime()
(2022, 11, 16, 11, 30, 55, 1, 320)
>>> time.gmtime(time.time())
(2022, 11, 16, 11, 31, 19, 2, 320)
```
I know that the Pico's RTC has an off-by-one on its day of the week, but that's a different issue
